dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -721,17 +721,36 @@ public boolean isUnmodifiableByOtherCode() {
         }
     }
 
+    /** FlowExpression representing a local variable. */
     public static class LocalVariable extends Receiver {
```

Comment:
Why do this only for this kind of Receiver? There are several other `hashCode` methods in the Receiver hierarchy that also look expensive. (I assume that this PR is a performance optimization.)
Would an alternative be to introduce the `hashcode` field in `Receiver`, override `hashCode` in `Receiver` to return that field, and rename the existing `hashCode` methods to `hashCodeCompute` or some such and use that method in the constructor?

Reply:
Yes, this PR is a possible performance optimization. (I should have included that in the description.)  I'm not sure if the performance improvement of this is worth the increase in memory.  Is this something @nhat-nguyen and you would like to look into?  If not, then we can just close this pull request.